packages/beacon-node/src/chain/bls/interface.ts

```diff
@@ -22,6 +22,14 @@ export type VerifySignatureOpts = {
   priority?: boolean;
 };
 
+export interface QueuedVerificationOpts extends VerifySignatureOpts {
+  /**
+   * Include multiplying in randomization factor for verification. Only applicable for verification
+   * of same message batches.  Applied via the worker pool options
+   */
+  disableSameMessageVerificationRandomness?: boolean;
```

Comment:
shouldn't this always be false else is inherently insecure? do we need this option or can it go away entirely?

Reply:
Thats a good point!.  For now it's a hidden flag and we should probably remove in a future update.

Reply:
imo it should be removed now.
we don't have a urgent need to push this out ASAP, worth getting it right

Reply:
OK.  Updated.